src/js/components/search/filters/location/CountryType.jsx

```diff
@@ -0,0 +1,44 @@
+/**
+ * CountryType.jsx
+ * Created by Emily Gullo 11/28/2016
+ **/
+
+import React from 'react';
+
+const propTypes = {
+    toggleCountry: React.PropTypes.func
+};
+
+export default class CountryType extends React.Component {
+
+    render() {
+        return (
+            <div className="location-radio">
+                <p>Show Only:</p>
+                <input
+                    type="radio"
+                    id="location-all"
+                    name="location"
+                    value="all"
+                    defaultChecked="true"
```

Comment:
Can the radio buttons be controlled by the LocationSearchContainer? Controlled components have their selection/checked value set by a parent component's state. When a change event occurs, it is bubbled up to the controlling component (as you currently do), but that parent component then pushes down the new selection/check value to the radio button, rather than allowing the browser to set it natively.

Take a look at the refactored award type filter components for an example of how it's done. Controlled components are important because there are multiple places where the filters can be set (such as the sticky top filter bar). By controlling selection and input values centrally, they can all remain in sync.